01/5_switch_statement/main.c

```diff
@@ -0,0 +1,23 @@
+#include <stdio.h>
+
+int main(void)
+{
+    int x = 3;
+    switch (x) {
+    case 0:
+        printf("Case 0\n");
+        break;
+    case 1:
+        printf("Case 1\n");
+        break;
+    case 2:
+        printf("Case 2\n");
+        break;
+    case 3:
+        printf("Case 3\n");

```

Comment:
no break? ignore me if it's on purpose

Reply:
this is on purpose to show it would fall through (though adding comment might be a good idea

Reply:
Was gonna suggest to just explain it in codeslides but... I guess you can comment on it as well...